lib/global_id/locator.rb

```diff
@@ -151,6 +158,10 @@ def initialize(block)
         def locate(gid)
           @locator.call(gid)
         end
+
+        def locate_many(gids, options = {})
+          gids.map { |gid| locate(gid) }
```

Comment:
Doing something like https://github.com/rails/globalid/pull/65/files#r28197892 would also make this faster for people who write a `BlockLocator`.